packages/api/src/controllers/webhook.ts

```diff
@@ -345,4 +279,93 @@ app.delete("/", authorizer({}), async (req, res) => {
   res.end();
 });
 
+const requestsFieldsMap: FieldsMap = {
```

Comment:
Do you think this `/requests` sub-path is going to grow a lot? If so, it might be worth considering exporting a separate `Router()` on another file, then using it here only with `app.use('/:id/requests', requests)`

Not a required change though, mostly a nit

Reply:
yep that sounds good, i've branched off this branch for the resend api so i'll implement this there to avoid merge conflicts